BloomFilter.md

```diff
@@ -132,10 +132,12 @@ to default row group size. It is also recommended to expose the ability for sett
 parameters to end users.
 
 #### File Format
-The Bloom filter data of a column chunk, which contains the size of the filter in bytes, the
-algorithm, the hash function and the Bloom filter bitset, is stored near the footer. The Bloom
-filter data offset is stored in column chunk metadata. Here are Bloom filter definitions in
-thrift:
+
+Each multi-block Bloom filter is required to work for only one column chunk. The data of a multi-block Bloom
+filter contains the header of one Bloom filter, which must include the size of the filter in bytes, the algorithm,
+the hash function, the compression and the Bloom filter bitset. The offset in column chunk metadata points to the start of
+the Bloom filter header. 
```

Comment:
This wording is strange because the header doesn't include the bloom filter bitset. I think you should change this to "The data of a multi-block bloom filter consists of the bloom filter header followed by the bloom filter bitset. The bloom filter header encodes the size of the bloom filter bit set in bytes that is used to read the bitset."

No need to call out the other header fields.

Reply:
Sound good. Updated.

Reply:
@rdblue, ready for review now.